system/xmllib/libxml_xpathprocessor.cpp

```diff
@@ -908,8 +908,9 @@ void copyXmlNode(IPropertyTree *tree, xmlNodePtr node)
 {
     for(xmlAttrPtr att=node->properties;att!=nullptr; att=att->next)
     {
+        VStringBuffer name("@%s", (const char *)att->name);
```

Comment:
I don't have a problem with this as-is, but wondering if it was changed to facilitate debugging?

Reply:
This is code that I added in a prior PR but hadn't used yet.  It ended up turning attributes into elements.  This fixes that and copies attributes as attributes.  I wish it wasn't necessary but ptree requires the @ sign.